flink-table/flink-sql-client/src/test/resources/sql/set.q

```diff
@@ -104,3 +104,70 @@ Was expecting one of:
     "," ...
 
 !error
+
+# test reset remove key
+reset execution.max-idle-state-retention;
+[WARNING] The specified key is not supported anymore.
+!warning
+
+set execution.max-table-result-rows=100;
+[WARNING] The specified key 'execution.max-table-result-rows' is deprecated. Please use 'sql-client.execution.max-table-result.rows' instead.
+[INFO] Session property has been set.
+!warning
+
+# test reset the deprecated key
+reset execution.max-table-result-rows;
+[WARNING] The specified key 'execution.max-table-result-rows' is deprecated. Please use 'sql-client.execution.max-table-result.rows' instead.
+[INFO] Session property has been reset.
+!warning
+
+set;
+execution.attached=true
+execution.savepoint.ignore-unclaimed-state=false
+execution.shutdown-on-attached-exit=false
+execution.target=remote
+jobmanager.rpc.address=$VAR_JOBMANAGER_RPC_ADDRESS
+pipeline.classpaths=
+pipeline.jars=$VAR_PIPELINE_JARS
+rest.port=$VAR_REST_PORT
+!ok
+
+
+set parallelism.default=3;
+[INFO] Session property has been set.
+!info
+
+# test reset option key
+reset parallelism.default;
```

Comment:
reset `execution.parallelism`

Reply:
@fsk119  This is test when `set parallelism.default`  deprecated key , if `reset parallelism.default` is work.

Reply:
emmm. My suggestion is to set option but reset deprecated key. But this case is to set option but reset option.

But I think the case above is enough to resolve the comments.

```
set execution.max-table-result-rows=200;
reset sql-client.execution.max-table-result.rows;
```

Reply:
oh~, i seem to have wrong , yes  you'r right